spec/models/automate_inflector_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe AutomateInflector do
+  before do
+    stub_const('ManageIQ::Providers::MegaCorp::CloudManager', Class.new(ManageIQ::Providers::CloudManager))
+    stub_const('ManageIQ::Providers::MegaCorp::CloudManager::Vm', Class.new(ManageIQ::Providers::CloudManager::Vm))
```

Comment:
this stubbing seems to break other tests :(
looks like I have to use an actual provider class here